app/controllers/admin/invoices_controller.rb

```diff
@@ -1,7 +1,21 @@
 class Admin::InvoicesController < ApplicationController
   def index
+    @invoices = Invoice.all
   end
 
   def show
+    @invoice = Invoice.find(params[:id])
+  end
+
+  def update
+    invoice = Invoice.find(params[:id])
+    invoice.update(invoice_params)
+    redirect_to admin_invoice_path(invoice.id)
```

Comment:
What happened to the conditional in this update or was that only for merchant/invoices?

Reply:
It's just not in here. There's an update conditional for admin merchants. This is admin invoices, so there might be one in merchant invoices

Reply:
That conditional was doing something that was already being done on line 12. I had Chris take a look at it office hours and said that we over coded it.

Reply:
Oh, awesome! Good to know.  I still need to do it though, I think, because i'm nested.